### ISSUE TITLE
markdown miss-hightlight "bash" when there's the word "until"

### ISSUE THREAD

**Issue by bric3** [atom/atom#5664]
I was writing a simple git documentation for some colleagues, with bash hightlighted snippet, but Atom miss-highlight if there's the `until` word and only with the bash style.

```md
    ```bash
    git log --since="1 week ago" --until="yesterday"
    ```
```

Changing the word a little (adding/removing a letter) highlighting is correct.

See : 

Incorrect :
![screenshot 2015-02-21 at 15 35 06](https://cloud.githubusercontent.com/assets/803621/6314442/5a0a32b2-b9df-11e4-9272-39f067968a62.png)

Correct (but changing one letter of the `until` letter):
![screenshot 2015-02-21 at 15 35 32](https://cloud.githubusercontent.com/assets/803621/6314444/72d286e6-b9df-11e4-92a0-8947549f1894.png)

---------------------------------
**OS** : OSX 10.10.2 Yosemite
**Atom** : 0.182.0

**Comment by izuzak** [comment]
This issue was moved to atom/language-gfm#78

**Comment by bric3** [comment]
atom/language-shellscript#52 has been fixed by atom/language-shellscript#65. Still needs to close atom/language-gfm#78 though.

Thanks again @50Wliu
